setup/pi/setup-teslausb

```diff
@@ -48,7 +48,7 @@ function read_setup_variables {
   BRANCH=${BRANCH:-main-dev}
   CONFIGURE_ARCHIVING=${CONFIGURE_ARCHIVING:-true}
   UPGRADE_PACKAGES=${UPGRADE_PACKAGES:-false}
-  TESLAUSB_HOSTNAME=${TESLAUSB_HOSTNAME:-teslausb}
+  export TESLAUSB_HOSTNAME=${TESLAUSB_HOSTNAME:-teslausb}
```

Comment:
I think that  "teslausb" should be "$HOSTNAME" or "$(hostname)", so that the current host name is used (in case the user changed it manually)

Reply:
But the user would change the TESLAUSB_HOSTNAME in the config file before installing the system. You use that to set the HOSTNAME during install - correct?
If they manually change the host name inside of the linux system (after install), they should change the config file sitting in /root to have the correct TESLAUSB_HOSTNAME as well as their save config file on their PC.
Or are you saying that the default of teslausb should be the HOSTNAME?  I am unsure how your one step setup handles that. Does the image have a default of teslausb when it installs and you change it to whatever is specified in the config file (if specified)?

Reply:
The host name is initially set by rc.local during early setup. setup-teslausb may update the host name again if it is later changed in the config file.
On second thought your change is fine though.

Reply:
I tested it with it set as follows:
export TESLAUSB_HOSTNAME=${TESLAUSB_HOSTNAME:-$(hostname)}

It seems to work but I don't know how it will do during an install on a new Pi.